src/components/ClusterManager.js

```diff
@@ -353,24 +344,16 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
           disabled: isDisabled
         }, [
           div({ style: { marginLeft: '0.5rem', paddingRight: '0.5rem', color: colors.dark() } }, [
-            div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Notebook Runtime'),
+            div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Application Compute'),
```

Comment:
We're not changing this text until we launch RStudio support, right?

Reply:
Correct, fixed thanks!